src/test/java/seedu/address/model/person/FindCommandPredicateTest.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+
+/**
+ * Contains equality test for FindCommandPredicate
+ */
+class FindCommandPredicateTest {
+
+    @Test
```

Comment:
Should there be a negative test case together with the positive test case in equals?